backend/public-graph/graph/resolver.go

```diff
@@ -785,39 +787,39 @@ func (r *Resolver) HandleErrorAndGroup(ctx context.Context, errorObj *model.Erro
 
 	if projectID == 1 {
 		if errorObj.Event == `input: initializeSession BillingQuotaExceeded` || errorObj.Event == `BillingQuotaExceeded` || errorObj.Event == `panic {error: missing operation context}` || errorObj.Event == `input: could not get json request body: unable to get Request Body unexpected EOF` || errorObj.Event == `no metrics provided` || errorObj.Event == `input: pushMetrics no metrics provided` {
```

Comment:
@mayberryzane we could also add some more patterns here to clear out the already-queued error messages?
```public-graph graphql request failed	1	2023-03-22 00:32:03.279896 +00:00	316
Error updating error group: Filtering out noisy error	1	2023-03-22 00:32:01.495118 +00:00	131
invalid metrics payload []	1	2023-03-22 00:33:01.350255 +00:00	95
error processing main session: error scanning session payload: error fetching events from Redis: error processing event chunk: The payload has an IncrementalSnapshot before the first FullSnapshot	1	2023-03-22 00:32:00.605651 +00:00	85
session has reached the max retry count and will be excluded: error scanning session payload: error fetching events from Redis: error processing event chunk: The payload has an IncrementalSnapshot before the first FullSnapshot	1	2023-03-22 00:32:00.498682 +00:00	85
```